openlibrary/templates/site/body.html

```diff
@@ -5,7 +5,7 @@
 $ show_banners = bodyid != 'form'
 $ bodyattrs = ctx.get('bodyattrs', [])
 
-<body id="$bodyid" class="$(' '.join(bodyclass))" $:(' '.join(bodyattrs))>
+<body class="$(' '.join(bodyclass))" $:(' '.join(bodyattrs))>
```

Comment:
Not this easy I'm afraid :)

Run `make css` and you'll see some CSS regressions on quite a few pages.

There are various CSS selectors e.g. `body#plain` that need to be fixed.

The good news is you can remove most of these, e.g. `body#plain div#headerLogo` becomes `div#headerLogo` but be careful about any generic selectors. `body#plain div` should not become `div`. 

Note we have page stylesheet entry points so a rule `body#plain div` can be moved into static/css/page-plain.less` if necessary.